tests/pyccel/test_pyccel.py

```diff
@@ -1189,3 +1189,8 @@ def test_time_execution_flag():
     assert 'Total' in result_lines[-2]
     for l in result_lines[1:-1]:
         assert ' : ' in l
+
+#------------------------------------------------------------------------------
+def test_char(language):
+    types = str
+    pyccel_test("scripts/c_arrays.py", language = language, output_dtype = types)
```

Comment:
Did you forget to commit the file `scripts/c_arrays.py` ?

Reply:
I did not because it already existed and I didn’t make any chances to it.

Reply:
Ah you are right sorry. It is already tested here:
https://github.com/pyccel/pyccel/blob/b39fe9a411736a833d980674b7f6380278febf0f/tests/pyccel/test_pyccel.py#L751-L755

In that case it seems that you have not finished adding a test as nothing new is tested.

Reply:
Ok, sounds good! Do I need to do anything else for you to approve my PR?

Reply:
You need to:
- add a test/multiple tests that tests the things that the PR is supposed to implement (so a test which translates a file with a char type annotation)
- Ensure all the tests pass (e.g. the Codacy static analysis is currently failing)
- Interact with the bot to get and complete your checklist
- Get the example to work. Your fix is not sufficient to make the following code work:
```python
if __name__ == '__main__':
        string = np.empty((2, 5), dtype='c')
        string[0] = "HELLO"
        string[1] = "WORLD"
        for i in range(5):
            print(string[0,i], end='')
```


The issue you are trying to fix specifically mentions this test: [pyccel/tests/internal/scripts/lapack/ex1.py](https://github.com/pyccel/pyccel/blob/e248f8a67adc06c079629924cec54279d6e7a709/tests/internal/scripts/lapack/ex1.py#L46) and asks that the header file describing the LAPACK library be updated to use `char` instead of `str` where appropriate.

Reply:
Oh sorry I misread the previous message.